.editorconfig

```diff
@@ -12,5 +12,5 @@ indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[*.scss]
+[*.{scss,md}]
```

Comment:
No reformatting needed…?

Reply:
No, the linter is already set correctly (https://github.com/react-ui-org/react-ui/blob/ea1d90341f80be267e82bb1ac59a3d1982e47e15/.markdownlint.jsonc#L6). This is here to make my IDE use correct indent when I press `Enter`.